examples/SEP-24/src/main/kotlin/org/stellar/walletsdk/Main.kt

```diff
@@ -0,0 +1,122 @@
+package org.stellar.walletsdk
+
+import java.math.BigDecimal
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import org.stellar.sdk.KeyPair
+import org.stellar.sdk.Network
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+// Setup main account that will fund new (user) accounts. You can get new key pair and fill it with
+// testnet tokens at
+// https://laboratory.stellar.org/#account-creator?network=test
+private val myAddress = "GCNVZJLJ5XZZALLZBR7BGUMGMF2KVZTBIGTU2A5KOPS2PPCZS6PV4LAD"
+private val myKey = "SDYGC4TW5HHR5JA6CB2XLTTBF2DZRH2KDPBDPV3D5TXM6GF7FBPRZF3I"
+
+suspend fun main() {
+  // Create instance of server that allows to connect to Horizon
+  val server = Server("https://horizon-testnet.stellar.org")
+  val wallet = Wallet()
+  // Generate new (user) account and fund it with 10 XLM from main account
+  val account = KeyPair.random()
+  val tx = wallet.fund(myAddress, account.publicKeyString, "10")
+
+  // Sign with your main account's key and send transaction to the network
+  println("Registering new account")
+  tx.sign(KeyPair.fromSecretSeed(myKey))
+  assert(wallet.submitTransaction(tx))
+
+  val anchor = Anchor(server, Network.TESTNET)
+
+  // Get info from the anchor server
+  val info = anchor.getInfo("https://testanchor.stellar.org/.well-known/stellar.toml")
```

Comment:
Passing only the home domain `testanchor.stellar.org` will work. We're appending `.well-known/stellar.toml` internally. Should we update description to make it more clear?

Reply:
I think so. 
IMO it's a good idea to pass home domain to Anchor so we can just call anchor.info